fixpatches/src/commonMain/kotlin/com/saveourtool/sarifutils/net/UrilUtils.kt

```diff
@@ -0,0 +1,176 @@
+@file:JvmName("UriUtils")
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
```

Comment:
This file looks like a candidate for you the new library

Reply:
Yes.

But let's postpone this till Monday.

Reply:
Referenced from #40, marking as resolved for now.